apps/indexer/ponder.schema.ts

```diff
@@ -124,16 +128,15 @@ export const transfer = onchainTable(
   }),
   (table) => ({
     pk: primaryKey({
```

Comment:
The relation mapping is incorrect. You're trying to match `votingPowerHistory.deltaMod` with `transfer.amount`, but `deltaMod` is described as 'non-negative' while `amount` could be any transfer amount. These fields represent different concepts and shouldn't be used as foreign key relations.